lib/process-data-availability.js

```diff
@@ -57,8 +60,8 @@ const getAvailabilityTemplate = () => {
       fields: getAvailabilityTotalsTemplate(),
       lookups: getAvailabilityTotalsTemplate(),
       resources: {},
-      resourcesBinary: {},
-    },
+      resourcesBinary: {}
```

Comment:
We should rename this at some point - it's not really accurate. Probably something like `resourceTotals` instead. But not anything we need to worry about now.